app/models/download.rb

```diff
@@ -4,9 +4,7 @@ class Download
   def initialize(work, options = {})
     @work = work
     @file_type = set_file_type(options.slice(:mime_type, :format))
-    # TODO: Our current version of the mime-types gem doesn't include azw3, but
-    # the gem cannot be updated without updating rest-client
-    @mime_type = @file_type == "azw3" ? "application/x-mobi8-ebook" : MIME::Types.type_for(@file_type).first
+    @mime_type =  @file_type == "azw3" ? "application/x-mobi8-ebook" : Marcel::MimeType.for(extension: @file_type).to_s
```

Comment:
We can sidestep the Rubocop by just removing the first bit (I ran one of the Cucumber tests to make sure and it still works)
```suggestion
    @mime_type = Marcel::MimeType.for(extension: @file_type).to_s
```